LayoutTests/imported/w3c/web-platform-tests/scroll-animations/crashtests/scroll-timeline-completion-crash.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<title>This test passes if it does not crash</title>
+<link rel="help" src="https://drafts.csswg.org/scroll-animations-1/#scroll-timeline-shorthand">
+<link rel="help" src="https://drafts.csswg.org/css-conditional-5/#container-queries">
```

Comment:
I don't see any uses of container queries, can you remove this link?

Reply:
Also the specific scroll-animations link is not really relevant. I think you can just link to `https://drafts.csswg.org/scroll-animations-1/`.

Reply:
Fixed.